lib/sdf-server/src/server/service/diagram/delete_connection.rs

```diff
@@ -43,15 +45,68 @@ pub async fn delete_connection(
     )
     .await?;
 
-    let from_component_schema =
-        Component::schema_for_component_id(&ctx, request.from_component_id).await?;
+    let from_component = Component::get_by_id(&ctx, request.from_component_id).await?;
 
-    let to_component_schema =
-        Component::schema_for_component_id(&ctx, request.to_component_id).await?;
+    let to_component = Component::get_by_id(&ctx, request.to_component_id).await?;
 
     let output_socket = OutputSocket::get_by_id(&ctx, request.from_socket_id).await?;
     let input_socket = InputSocket::get_by_id(&ctx, request.to_socket_id).await?;
 
+    let base_change_set_ctx = ctx.clone_with_base().await?;
+
+    let base_from_component = Component::get_by_id(&base_change_set_ctx, request.from_component_id)
+        .await
+        .ok();
+    let base_to_component = Component::get_by_id(&base_change_set_ctx, request.to_component_id)
+        .await
+        .ok();
```

Comment:
These should use `try_get_by_id` instead, since it returns a `ComponentResult<Option<Self>>`. While the frontend can't do anything about any errors that happen, we shouldn't hide them from our monitoring.